src/Bridge/Doctrine/Orm/SubresourceDataProvider.php

```diff
@@ -195,4 +122,77 @@ public function getSubresource(string $resourceClass, array $identifiers, array
 
         return $context['collection'] ? $query->getResult() : $query->getOneOrNullResult();
     }
+
+    /**
+     * @throws RuntimeException
+     */
+    private function buildQuery(array $identifiers, array $context, QueryNameGenerator $queryNameGenerator, QueryBuilder $previousQueryBuilder, string $previousAlias, int $remainingIdentifiers, QueryBuilder $topQueryBuilder = null): QueryBuilder
+    {
+        if (!$remainingIdentifiers) {
```

Comment:
`0 < --$remainingIdentifiers` then we can avoid `--` below and the `$remainingIdentifiers - 1` operation?

Reply:
I think it's clearer this way. Because if `--` is done here it's more the position of the identifier and not the remaining ones. And you have to add the operation `$remainingIdentifiers + 1` where there is `$remainingIdentifiers`. WDYT?

Reply:
ok then maybe `0 === $remainingIdentifiers` instead of `!`?

Reply:
If you prefer. Then not `0 >= $remainingIdentifiers`?

Reply:
Or even `$remainingIdentifiers <= 0` (I don't really like Yoda style for comparison)?

Reply:
Dunno it's fine like this as well.